dev/devicelab/lib/tasks/dart_plugin_registry_tests.dart

```diff
@@ -31,46 +31,46 @@ TaskFunction dartPluginRegistryTest({
             'io.flutter.devicelab',
             '--platforms',
             'macos',
-            'plugin_platform_implementation',
+            'aplugin_platform_implementation',
```

Comment:
@stuartmorgan I think this is what you meant when you said use 'aplugin', can you confirm?

Reply:
Yes, that's correct.